astro/src/content/blog/fusionauth-re-invent-2023.mdx

```diff
@@ -0,0 +1,59 @@
+---
+publish_date: 2023-11-27
+title: "We'll see you at AWS re:Invent 2023"
+description: FusionAuth is going to AWS re:Invent. Stop by and see us!
+authors: Alex Patterson
+image: /img/blogs/fusionauth-re-invent-2023/fusionauth-template.png
+categories: Events
+tags: events, reinvent, conference
+excerpt_separator: "{/* more */}"
+---
+
+AWS re:Invent is one of the largest technical conferences in the world. In the cloud space, it is the go-to event. This week there will be tens of thousands of attendees in Las Vegas, Nevada. The [FusionAuth team will be among them](/aws-reinvent23), and we'd love to see you there!
+
+{/* more */}
+
+## Why the team is attending
+
+We're going to re:Invent for many reasons.
+
+One is that FusionAuth and our clients often deploy on the AWS public cloud. Some clients use vanilla EC2 instances with RDS, others deploy on solutions like EKS and Fargate (yes, [FusionAuth supports Kubernetes](/docs/v1/tech/installation-guide/kubernetes/fusionauth-deployment)).
+
+AWS re:Invent is an opportunity to meet developers where they are both during talks and in the ever-present hallway track. We look forward to exchanging knowledge and ideas with folks building on the AWS platform.
+
+We're also excited to chat with attendees about authentication. We like to hear from developers about what they are building on top of solutions like FusionAuth, Auth0 and Cognito.
+
+You can migrate from [Auth0](/docs/v1/tech/migration-guide/auth0) to FusionAuth and we're of course happy to help. Migrating away from Cognito isn't as straightforward, but [there are ways](/docs/v1/tech/migration-guide/cognito).
+
+We're also looking forward to some AWS re:Invent sessions. 
+
+Exciting ones include 
+- SEC209 | Modernize authorization: Lessons from cryptography and authentication
+- DAT333 | Building highly resilient applications with Amazon DynamoDB
+- SEC322 | Integrate apps with Amazon Cognito and Amazon Verified Permissions
+- BOA209 | Build verifiable and effective application authorization in 40 minutes
+
+Of course, it isn't all education. There's also fun to be had!
+
+One fun thing to do is join [Corey Quinn's annual nature walk](https://www.lastweekinaws.com/blog/aws-re-invent-what-you-actually-need-to-know-before-you-go/) where he skewers Expo vendors. This is happening on Wednesday at 1pm.
+
+There are also numerous parties. If you are attending AWS, chances are at least one vendor has emailed you with an invite. This Twitter account, [@reInventParties](https://twitter.com/reInventParties/), is a great way to discover additional opportunities for fun.
```

Comment:
Should we mention the FusionAuth party here?  It may be more invite only so, maybe not.  Just want to make sure you thought about it.